app/src/main/java/cs465/illinois/edu/dogonthequad/ExpandableListAdapter.java

```diff
@@ -0,0 +1,106 @@
+package cs465.illinois.edu.dogonthequad;
+
+import java.util.HashMap;
+import java.util.List;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseExpandableListAdapter;
+import android.widget.TextView;
+
+public class ExpandableListAdapter extends BaseExpandableListAdapter {
```

Comment:
What is the purpose of this class? (This isn't a criticism, by the way. I'm just not familiar with coding expandable lists.)

Reply:
The general standard in android for lists (expandable lists, recycler views, etc.) is to have an abstract base class and the app developer implements an adapter for it, which can handle things like populating list entries with data, touch events, etc.

Reply:
Thanks!